package.json

```diff
@@ -449,17 +449,17 @@
         },
         {
           "command": "maven.project.excludeDependency",
-          "when": "view == mavenProjects && viewItem =~ /^(Dependency)|(conflictDependency)$/",
+          "when": "view == mavenProjects && viewItem =~ /^(?!.*?\\+root)/",
```

Comment:
will the command show on other non-dependency node?

Reply:
@jdneo 
Here we have these node types: `maven:dependency[+root/+conflict]`. Is there a regexp to query "all dependency nodes with no +root property"? (I assume there should be a negative lookahead mechanism)

@Melody618  If the answer is no, we can also workaround it, by adding `+nonRoot` to non-root nodes.

Reply:
Try `maven:dependency(?=.*?\b\+conflict\b)(?!.*?\b\+root\b)`

Reply:
![image](https://user-images.githubusercontent.com/2351748/129995954-11bd3bb9-93c2-4416-8325-db072ad7a60b.png)

Cool.

@Melody618 please use above regexp.
@jdneo suggest to add `(?!.*?\b\+exclude_prop\b)` in Wiki.

Reply:
as the picture shows, the regexp is not able to hit `maven:dependency`

Reply:
of course it's as expected. for your case, simply append `(?!.*?\b\+root\b)`.